### DISCUSSION TITLE
リポジトリの分け方について

### DISCUSSION THREAD

**Discussion by doranko** [gBenkyous/DevStudyHub#4] [Q&A]
@honmaharuki 　CC: @watabusi @gonzaresu-sengoku 
wiki リポジトリと別にリポジトリを切っている目的は何でしょうか？
inTime限定ではない汎用的な知識に関する内容を残す目的かと思うのですが、wikiリポジトリでも良いのではないかと思いました。

githubで一元管理する最大の目的は「どこに記録したのかを探さなくてよくする」ということだと認識しているのですが、
現状ですと「wikiに書いてるの？DevStudyHubに書いてるの？inTimeのissueだっけ？」となるかと思います。
一旦wikiで管理してどうしても煩雑に感じて分けたいという要望が出てから切り分けるのでも良いような気がしますが如何でしょうか？

**Comment by haruki256** [comment]
こちらについてはMTGにて共有
内容が機密要素を含んでいるため記載省略
